ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/WearApp.kt

```diff
@@ -503,6 +512,29 @@ fun WearApp(
                     FullScreenProgressIndicator()
                 }
 
+                composable(
+                    route = Screen.Theme.route,
+                    arguments = listOf(
+                        // In this case, the argument isn't part of the route, it's just attached
+                        // as information for the destination.
+                        navArgument(SCROLL_TYPE_NAV_ARGUMENT) {
+                            type = NavType.EnumType(DestinationScrollType::class.java)
+                            defaultValue = DestinationScrollType.SCALING_LAZY_COLUMN_SCROLLING
+                        }
+                    )
+                ) { it ->
+                    val scalingLazyListState = scalingLazyListState(it)
+                    val focusRequester = remember { FocusRequester() }
+
+                    ThemeScreen(
+                        scalingLazyListState = scalingLazyListState,
+                        focusRequester = focusRequester,
+                        currentlySelectedColors = themeColors,
+                        availableThemes = themeValues
+                    ) { colors -> themeColors = colors }
```

Comment:
Curious: is there a agreed style for event handlers as last method param? named vs trailing lambda?

Reply:
I think this falls under the general callback as last property to allow for lambdas unless it causes confusion. But I agree with compose where we will often have a content lambda at the end we will often end up naming callbacks. I think the basic premise should be considering have last param as something lambda'erable if you can.